lib/InitAll.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include "torch-mlir/InitAll.h"
 
+#include <mlir/Dialect/Func/Extensions/InlinerExtension.h>
```

Comment:
Please match the quote format of the other includes.

Reply:
pushed a commit for the same